backends/openvino/README.md

```diff
@@ -43,8 +43,8 @@ executorch
 Before you begin, ensure you have openvino installed and configured on your system:
 
 ```bash
-git clone https://github.com/openvinotoolkit/openvino.git
-cd openvino && git checkout releases/2025/1
+git clone https://github.com/daniil-lyakhov/openvino.git
+cd openvino && git checkout dl/executorch/yolo12
```

Comment:
is this meant to be left like this?

Reply:
We are working on a [fix ](https://github.com/openvinotoolkit/openvino/pull/30557) in the openvino itegration, I'll update the branch as soon as we will have it merged

Reply:
Updated